sw/device/tests/sim_dv/BUILD

```diff
@@ -231,3 +231,23 @@ opentitan_functest(
         "//sw/device/lib/testing:sram_ctrl_testutils",
     ],
 )
+
+opentitan_functest(
+    name = "adc_ctrl_sleep_debug_cable_wakeup_test",
+    srcs = ["adc_ctrl_sleep_debug_cable_wakeup_test.c"],
+    targets = ["dv"],
+    deps = [
+        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+        "//sw/device/lib/base:mmio",
+        "//sw/device/lib/dif:adc_ctrl",
+        "//sw/device/lib/dif:pwrmgr",
+        "//sw/device/lib/dif:rstmgr",
+        "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing:pwrmgr_testutils",
+        "//sw/device/lib/testing:rstmgr_testutils",
+    ],
+)
+
+
```

Comment:
Where'd all this whitespace come from? (I assume the whitespace checker catches this, yes?)